client/src/components/EditMyProfile.js

```diff
@@ -0,0 +1,98 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { Formik, Form } from 'formik';
+import { ProfileContext } from '../context/ProfileContext';
+import * as Yup from "yup";
+import FormField from "../constant/FormField";
+import StyledButton from '../constant/StyledButton';
+import avatar from '../assets/icons/avatar.svg';
+import GraduateForm from './GraduateForm';
+
+const EditMyProfile = () => {
+	const { profile, editProfile, manageEdit, edit }  = useContext(ProfileContext);
+
```

Comment:
You can use better naming conventions:
`profile`/`setProfile`
`edit`/`setEdit` but I would probably make it more clear this is a boolean by naming it instead something like `isEditing`/`setIsEditing` or `editing`/`setEditing`
also instead of `profiles` I would use `allProfiles` (/`setAllProfiles`), it's much easier to see it's for all of them.

This way anyone looking at your code will know straight away what are states and what are actions.